modules/microshift-adding-repos-to-image-builder.adoc

```diff
@@ -19,7 +19,7 @@ Use the following procedure to add the {product-title} repositories to Image Bui
 +
 [source,terminal,subs="attributes+"]
 ----
-$ sudo subscription-manager repos
-    --enable rhocp-{ocp-version}-for-{rhel-major}-$(uname -i)-rpms
+$ sudo subscription-manager repos \
```

Comment:
When using the copy text helper icon, on the right side of this example text, it includes the '$'.
When the user pastes this into their shell it results in an error.

`-bash: $: command not found`

This issue exists throughout the doc.

Reply:
@jogeo This is the correct docs style; we use the symbol to separate the "command prompt" from the _output_ of commands in docs. There is a Customer Portal issue around this that I am investigating with the DPM.